### PR TITLE
CORTX-33384 : TE RBF : F-40E : CSM : Deployment: Config: Resource Limits: CSM

### PR DESCRIPTION
**Pull request by RajeshPratapSInghChouhan**
# Problem Statement
[CORTX-33384](https://jts.seagate.com/browse/CORTX-33384) TE RBF : F-40E : CSM : Deployment: Config: Resource Limits: CSM

# Design
https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/1077084998/QA+F-40E+CSM+Resource+Limits+tests

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by RajeshPratapSInghChouhan** [comment]
inti run : https://eos-jenkins.colo.seagate.com/view/QA-R2/job/QA/job/QA_R2_Single_Runner_CSM_error_msg_check/115/console
TE : [TEST-44481](https://jts.seagate.com/browse/TEST-44481)

**Comment by RajeshPratapSInghChouhan** [comment]
tests are failing due to : [CORTX-33861](https://jts.seagate.com/browse/CORTX-33861)

**Comment by RajeshPratapSInghChouhan** [comment]
with updated code, test run :
https://eos-jenkins.colo.seagate.com/view/QA-R2/job/QA/job/QA_R2_Single_Runner_CSM_error_msg_check/116/console

File: tests/csm/rest/test_csm_users.py [hunk 1]
```diff
@@ -4656,7 +4656,6 @@ def test_28516(self):
         self.csm_obj.check_expected_response(response, HTTPStatus.OK)
 
 
-    @pytest.mark.skip(reason="not_in_main_build_yet")
```
Comment (kachhwahadivya) [inline tests/csm/rest/test_csm_users.py:4659]: Can u map defect ID? As everything is in main now

File: libs/csm/rest/csm_rest_parallel_operations.py [hunk 2]
```diff
@@ -111,6 +113,41 @@ def execute_max_user_loop(self, jmx_file, users, request_limit, ops):
         self.counter = 0
         return result
 
+    def execute_max_iam_user_create_loop(self, jmx_file, users, request_limit):
```
Comment (kachhwahadivya) [inline libs/csm/rest/csm_rest_parallel_operations.py:116]: Can u add logic to parallel delete

Comment (kachhwahadivya) [reply libs/csm/rest/csm_rest_parallel_operations.py:116]: We can plan follow ticket for this.

File: libs/csm/rest/csm_rest_parallel_operations.py [hunk 3]
```diff
@@ -113,6 +116,58 @@ def execute_max_user_loop(self, jmx_file:str, users:int, request_limit:int, ops:
...
+        jmx_file = "CSM_Create_N_IAM_Users_Loaded.jmx"
+        self.log.info("Running jmx script: %s", jmx_file)
+        request_limit = self.get_request_usage_limit()
+        result = self.execute_max_iam_user_loop(jmx_file, users, request_limit,
+                                                       ops = "create")
+        return result
+
+    def delete_multi_iam_user_loaded(self):
+        """
+        Create count number of CSM in parallel using jmeter libs
```
Comment (kachhwahadivya) [inline libs/csm/rest/csm_rest_parallel_operations.py:159]: delete

**Review by kachhwahadivya — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

File: libs/csm/rest/csm_rest_parallel_operations.py (hunk 3 above)
Comment (RajeshPratapSInghChouhan) [reply libs/csm/rest/csm_rest_parallel_operations.py]: updated

**Comment by RajeshPratapSInghChouhan** [comment]
#pytest --local True --collect-only
: 'tests/security/test_cortx_port_scanner_netstat.py::test_cortx_port_scanner_netstat', 'test_id': 'TEST-34218', 'marks': ['security']}] created at /root/jenkins/workspace/QA/QA_R2_Single_Runner_CSM_error_msg_check/cortx-test/log/te_meta.json
Successfully unmounted directory

========================================================================= 2418 tests collected in 8.60s ==========================================================================

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**